test/e2e/bind/happy-case_test.go

```diff
@@ -69,172 +70,280 @@ func testHappyCase(t *testing.T, scope kubebindv1alpha1.Scope) {
 	consumerConfig, consumerKubeconfig := framework.NewWorkspace(t, framework.ClientConfig(t), framework.WithGenerateName("test-happy-case-consumer"))
 	framework.StartKonnector(t, consumerConfig, "--kubeconfig="+consumerKubeconfig)
 
-	consumerClient := framework.DynamicClient(t, consumerConfig).Resource(
-		schema.GroupVersionResource{Group: "mangodb.com", Version: "v1alpha1", Resource: "mangodbs"},
-	).Namespace("default")
-	providerClient := framework.DynamicClient(t, providerConfig).Resource(
-		schema.GroupVersionResource{Group: "mangodb.com", Version: "v1alpha1", Resource: "mangodbs"},
-	)
+	serviceGVR := schema.GroupVersionResource{Group: "mangodb.com", Version: "v1alpha1", Resource: "mangodbs"}
+	if scope == kubebindv1alpha1.ClusterScope {
```

Comment:
I think we have two scopes now:
1. namespaced resource, but with cluster or namespace scoped informers.
2. cluster scoped resources, always with cluster scoped informers.

Reply:
I.e. we want to restore this test to verify both modes in (1).

Reply:
> I.e. we want to restore this test to verify both modes in (1).

Done in latest push, and also squashed into the commit that generalizes the e2e test.